setup.py

```diff
@@ -141,5 +142,12 @@ def run_tests(self):
         'clean': CleanCommand,
         'test': PyTestCommand,
     },
-    install_requires=requires,
+    install_requires=install_requires,
+    setup_requires=[
+        'wheel==0.29.0',
+    ],
+    tests_require=[
```

Comment:
I've had trouble getting travis to play well with tests_require, which is why mock and pytest are in test_reqs.txt instead.